Debian #377934; GNOME #436804; remove Ok and Cancel buttons from included
themes.

Index: gdm-2.18.1/gui/greeter/themes/circles/circles.xml
===================================================================
--- gdm-2.18.1.orig/gui/greeter/themes/circles/circles.xml	2007-04-09 07:15:09.000000000 +0200
+++ gdm-2.18.1/gui/greeter/themes/circles/circles.xml	2007-05-02 11:04:25.000000000 +0200
@@ -78,14 +78,6 @@
 	  </item>
 	</fixed>
       </item>
-      <item type="button" id="ok_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="ok"/>
-      </item>
-      <item type="button" id="cancel_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="startagain"/>
-      </item>
       <item type="label" id="pam-message">
         <pos anchor="n" x="50%"/>
         <normal color="#000000" font="Sans 12"/>
Index: gdm-2.18.1/gui/greeter/themes/happygnome/happygnome.xml
===================================================================
--- gdm-2.18.1.orig/gui/greeter/themes/happygnome/happygnome.xml	2007-04-09 07:15:09.000000000 +0200
+++ gdm-2.18.1/gui/greeter/themes/happygnome/happygnome.xml	2007-05-02 11:04:25.000000000 +0200
@@ -90,14 +90,6 @@
 	  </item>
 	</fixed>
       </item>
-      <item type="button" id="ok_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="ok"/>
-      </item>
-      <item type="button" id="cancel_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="startagain"/>
-      </item>
       <item type="label" id="pam-message">
         <pos anchor="n" x="50%"/>
         <normal color="#000000" font="Sans 12"/>
Index: gdm-2.18.1/gui/greeter/themes/happygnome-list/happygnome.xml
===================================================================
--- gdm-2.18.1.orig/gui/greeter/themes/happygnome-list/happygnome.xml	2007-04-09 07:15:08.000000000 +0200
+++ gdm-2.18.1/gui/greeter/themes/happygnome-list/happygnome.xml	2007-05-02 11:04:25.000000000 +0200
@@ -101,14 +101,6 @@
 	  </item>
 	</fixed>
       </item>
-      <item type="button" id="ok_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="ok"/>
-      </item>
-      <item type="button" id="cancel_button">
-        <pos anchor="n" x="50%" height="32" width="50%"/>
-        <stock type="startagain"/>
-      </item>
       <item type="label" id="pam-message">
         <pos anchor="n" x="50%"/>
         <normal color="#000000" font="Sans 12"/>
